html/log.php

```diff
@@ -0,0 +1,98 @@
+<?php
+header('Access-Control-Allow-Origin: *');
+header('Content-Type: application/json');
+
+function get_string_between($string, $start, $end){
+    $string = ' ' . $string;
+    $ini = strpos($string, $start);
+    if ($ini == 0) return '';
+    $ini += strlen($start);
+    $len = strpos($string, $end, $ini) - $ini;
+    return substr($string, $ini, $len);
+}
+
+$credits = "Made by AnotherPillow, base PHP code by xintrix (xintrix-dev on github)";
+
+$mc1 = get_string_between($link, '"log-message-text">', '</span>');
+
+$files = glob('../images/*');
+//$loglink = "https://smapi.io/log/1d07e685d20b4fa2be7907c9b4b3aeb3";
+$loglink = $_SERVER["QUERY_STRING"];
+$logid = ltrim($loglink,"https://smapi.io/log/");
+$link = file_get_contents($loglink);
+
+$sdv = get_string_between($link, 'data-game-version="', '"');
+//set success to false if sdv is blank
+if ($sdv == "") {

```

Comment:
We can combine these if/else checks into `$sdvsuccess = $sdv != ""`, both here and elsewhere.